great_expectations/experimental/metric_repository/column_descriptive_metrics_metric_retriever.py

```diff
@@ -109,6 +109,8 @@ def _get_table_row_count(
             computed_metrics=computed_metrics,
             aborted_metrics=aborted_metrics,
         )
+        if value is None:
+            self._raise_on_missing_metric(metric_name, exception)
```

Comment:
Is there a reason that OSS/Core can't just tell us if there's a connection issue? It seems weird that we need to check for missing table metrics to suggest to the user that there may be a connection issue.

Reply:
Really good question. Was just looking at that with @josectobar last night who first reported this. There is a test_connection() method on the data source that we can use but that did not catch the error he was seeing (we think it might be role / permissions related that crops up when you try to query data vs access table schema). Let me look into this more.

Reply:
Done in https://github.com/great-expectations/cloud/pull/91, and `test_connection()` will be modified in a followup PR to also test selecting data to make sure the user has select permissions.